src/main.py

```diff
@@ -34,7 +43,17 @@ def main() -> None:
     )
     menu.logo_image = pygame.image.load("../resources/graphics/logo.png")
 
+    track = Track.from_points(tracks[0]["points"])
+    sim = Simulation(track)
+    tv = TrackView(sim)
+
+    track1 = Track.from_points(tracks[1]["points"])
+    sim1 = Simulation(track1)
+    tv1 = TrackView(sim1)
+
     window.add_view(menu, 0, True)
+    window.add_view(tv, 1)
+    window.add_view(tv1, 2)
```

Comment:
In my opinion it's better to have more descriptive names: 'track_view' istead of 'tv'. Same for 'sim' etc.
Fortunately we don't have to pay for any additional letters in our code.

Reply:
XD

Reply:
You know it is just to make map preview easily accessible for testing and maps are going to be selected from options menu in final version?